src/App/Fossa/Config/Common.hs

```diff
@@ -138,6 +141,9 @@ data CacheAction
 defaultTimeoutDuration :: Duration
 defaultTimeoutDuration = Minutes 60
 
+readMWithError :: Read a => String -> ReadM a
+readMWithError errMsg = auto <|> readerError errMsg
```

Comment:
Do we have to provide a custom error message? I thought `auto` would generate one based on the type provided?

Reply:
It does, and the default isn't terrible but I chose this because the default doesn't actually tell you what's expected. Only that you did something wrong:
```
option --policy-id: cannot parse value `hello'
```

Reply:
Oh, gross. Yeah I like your option better.